lib/squasher.rb

```diff
@@ -18,7 +18,7 @@ def clean
 
   def rake(command, description = nil)
     tell(description) if description
-    system("bundle exec rake #{ command } 2>/dev/null")
+    system("bundle exec rake #{ command }")
```

Comment:
I doesn't look like Hound is following the same conventions as the current codebase...